packages/lib/src/vis/tiles/store.ts

```diff
@@ -1,4 +1,4 @@
-import create from 'zustand';
+import { create } from 'zustand';
```

Comment:
Note the difference between `create` and `createStore`. The former returns a custom hook, while the latter returns a vanilla store object. It's the latter that can be injected via a context and retrieved with `useStore(myVanillaStore)` (or `useStore(useContext(MyVanillaStoreContext)`). Here we use a simple global store, so we can use the hook API.